ground/gcs/src/plugins/uploader/tl_dfu.cpp

```diff
@@ -469,11 +472,19 @@ bool DFUObject::OpenBootloaderComs(USBPortInfo port)
         AbortOperation();
         if(!EnterDFU()) {
             TL_DFU_QXTLOG_DEBUG(QString("Could not process enterDFU command"));
+            if ((retries--) > 0) {
```

Comment:
Basically how I see the pro/con--  skipping the brackets saves some screen space.  But it makes it easy to group statements wrong accidentally and have really unexpected behavior.